config/database.yml

```diff
@@ -24,7 +24,8 @@ default: &default
 development:
   <<: *default
   database: webpage-versions-db_development
-
+  password: postgres
+  #password: Figaro.env.LOGINROLE_PASSWORD
```

Comment:
@Mr0grog sir i have committed some changes with option 1. They are working. I went ahead with 1 as it had less changes to make. However I am reading about the comparison between 1 and 2 and shall implement the better one as final changes